mLRS/CommonTx/esp.h

```diff
@@ -71,6 +71,9 @@ bool com_is_full(void)
 {
 #ifdef DEVICE_HAS_COM_ON_USB
     return usb_tx_full();
+#elif defined DEVICE_HAS_NO_COM
+    //return !uart_tx_notfull();
```

Comment:
is this a comment you've added and wasn't in the original code

I strugge a bit

Reply:
I added this and it needs to be cleaned up, and, depending on baud rates, uart_tx_notfull needs a real implementation.